docs_user/modules/con_block-storage-service-requirements.adoc

```diff
@@ -1,32 +1,16 @@
 [id="block-storage-requirements_{context}"]
 
-= Block Storage service requirements
+= {block_storage} requirements
 
-The Block Storage service (cinder) has both local storage used by the service and {rhos_prev_long} ({OpenStackShort}) user requirements.
-
-Local storage is used for example when downloading a {image_service_first_ref} image for the create volume from image operation, which can become considerable when having
-concurrent operations and not using the Block Storage service volume cache.
-
-In the Operator deployed {rhos_long}, there is a way to configure the
-location of the conversion directory to be an NFS share (using the extra
-volumes feature), something that needed to be done manually before.
-
-Even if it's an adoption and it may seem that there's nothing to consider
-regarding the Block Storage service backends, because you are using the same ones that you are
-using in your current deployment, you should still evaluate it, because it may not be so straightforward.
-
-First you need to check the transport protocol the Block Storage service backends are using:
-RBD, iSCSI, FC, NFS, NVMe-TCP, etc.
-
-Once you know all the transport protocols that you are using, you can make
-sure that you are taking them into consideration when placing the Block Storage services (as mentioned above in the Node Roles section) and the right storage transport related binaries are running on the {rhocp_long} nodes.
-
-Detailed information about the specifics for each storage transport protocol can be found in the xref:openshift-preparation-for-block-storage-adoption_storage-requirements[{OpenShiftShort} preparation for {block_storage} adoption].
-
-The recommended way to deploy {block_storage} volume back ends is to use a {block_storage} volume service for each back end.
-
-For example, you have an LVM and a Ceph back end and two entries in `cinderVolumes`, and you cannot set global defaults for all volume services. You must define a service for each of them:
+To prepare to adopt your {block_storage_first_ref}, review the following requirements:
```

Comment:
Are the items in this list required for the user to do before starting adoption? Or are these items more considerations or guidelines?

Reply:
I would consider this a list of things users need to go through, and potentially do, before they jump into doing the adoption.